src/mainwindow.cpp

```diff
@@ -5692,7 +5696,7 @@ void MainWindow::openLocalUrl(QString urlString) {
             Note note = Note::fetch(noteId);
             if (note.isFetched()) {
                 // set current note
-                setCurrentNote(note);
+                setCurrentNote(std::move(note));
```

Comment:
This, I think you should review.
after an `std::move(note)`, the `note` object will become empty, so it can't be used again in the function. It seems to me that after the first this `if`, it's not likely that we can fall into any other condition branch, so we are safe. but better be sure.

I can remove these in any case.

Reply:
Looks like it's safe. :smiling_imp: